examples/win-title

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Example for $XDG_CONFIG_HOME/nsxiv/exec/win-title
+# Called by nsxiv(1) on each redraw.
+# The output is set as nsxiv's window title.
+# Arguments, "Optional" arguments might be empty:
+#   $1: full path to the current image
+#   $2: current file number
+#   $3: total file number
+#   $4: image width (Optional: Disabled on thumbnails mode)
+#   $5: image height (Optional: Disabled on thumbnails mode)
+#   $6: current zoom (Optional: Disabled on thumbnails mode)
+
+exec 2>/dev/null
+
+filename="${1##*/}"
+
+if [ -n "$4" ]; then # image mode
+	echo "nsxiv - ${filename} | ${4}x${5} ${6}% [${2}/${3}]"
```

Comment:
`echo` outputs a trailing newline which is shown as some weird symbol on some systems (such as mine, using Polybar). `echo -n` is not POSIX. Don't think nsxiv should deal with invalid characters, so let's offload that here.
```suggestion
	printf "%s" "nsxiv - ${filename} | ${4}x${5} ${6}% [${2}/${3}]"
```

Also from echo(1p):
> New applications are encouraged to use printf instead of echo.

Reply:
+1, was about to comment this same issue. The trailing newline causes a box character to appear at the end of the title in herbstluftwm.

Reply:
Didn't realize the newline could cause trouble. Using `printf` is fine by me.